cpp/tests/utilities/generate_graph_from_edgelist.cu

```diff
@@ -30,39 +30,47 @@ namespace test {
 
 namespace detail {
 
-template <typename vertex_t,
-          typename edge_t,
-          typename weight_t,
-          bool store_transposed,
-          bool multi_gpu>
-std::enable_if_t<
-  multi_gpu,
-  std::tuple<
-    cugraph::experimental::graph_t<vertex_t, edge_t, weight_t, store_transposed, multi_gpu>,
-    rmm::device_uvector<vertex_t>>>
-generate_graph_from_edgelist(raft::handle_t const& handle,
-                             rmm::device_uvector<vertex_t>&& vertices,
-                             rmm::device_uvector<vertex_t>&& edgelist_rows,
-                             rmm::device_uvector<vertex_t>&& edgelist_cols,
-                             rmm::device_uvector<weight_t>&& edgelist_weights,
-                             bool is_symmetric,
-                             bool test_weighted,
-                             bool renumber)
-{
-  CUGRAPH_EXPECTS(renumber, "renumber should be true if multi_gpu is true.");
-
-  auto& comm               = handle.get_comms();
-  auto const comm_size     = comm.get_size();
-  auto const comm_rank     = comm.get_rank();
-  auto& row_comm           = handle.get_subcomm(cugraph::partition_2d::key_naming_t().row_name());
-  auto const row_comm_size = row_comm.get_size();
-  auto& col_comm           = handle.get_subcomm(cugraph::partition_2d::key_naming_t().col_name());
-  auto const col_comm_size = col_comm.get_size();
-
-  vertex_t number_of_vertices = static_cast<vertex_t>(vertices.size());
+template <typename vertex_t>
```

Comment:
Do you still need the changes in this file after the test updates?

Reply:
Not necessarily, I will verify this.  It does successfully allow supporting of generating a multi-gpu graph without renumbering - in case we find another need for that.

That can be useful for debugging algorithms (knowing the numbering a priori and that it remains consistent across different shape MG tests).

Reply:
I changed this file significantly in PR https://github.com/rapidsai/cugraph/pull/1443, and this is not necessary for testing, we may better undo this change for now.

Reply:
I will revert tomorrow.

Reply:
Done